auth/service_test.go

```
@@ -845,24 +853,28 @@ func TestIdentify(t *testing.T) {
+	repoCall = krepo.On("Save", mock.Anything, mock.Anything).Return(mock.Anything, nil)
 	exp1 := time.Now().Add(-2 * time.Second)
 	expSecret, err := svc.Issue(context.Background(), loginSecret.AccessToken, auth.Key{Type: auth.APIKey, IssuedAt: time.Now(), ExpiresAt: exp1})
 	assert.Nil(t, err, fmt.Sprintf("Issuing expired login key expected to succeed: %s", err))
-	saveCall.Unset()
+	repoCall.Unset()
 
-	repocall2 := krepo.On("Save", mock.Anything, mock.Anything).Return(mock.Anything, nil)
-	repocall3 := pEvaluator.On("CheckPolicy", mock.Anything, mock.Anything).Return(nil)
+	repoCall = krepo.On("Save", mock.Anything, mock.Anything).Return(mock.Anything, nil)
+	repoCall1 = pEvaluator.On("CheckPolicy", mock.Anything, mock.Anything).Return(nil)
+	repoCall2 = callback.On("Authorize", mock.Anything, mock.Anything).Return(nil)
```

golangci-lint (1.62.2)

SA4006: this value of `repoCall2` is never used

(staticcheck)